ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ApprovedDeferrals.tsx

```diff
@@ -170,34 +164,91 @@ function ApprovedDeferrals() {
                     </Tr>
                 </Thead>
                 <Tbody>
-                    {data.map((exception) => {
-                        const { id, name, requester, createdAt, scope } = exception;
-                        return (
-                            <Tr key={id}>
-                                <Td>
-                                    <RequestIDLink id={id} name={name} context="CURRENT" />
-                                </Td>
-                                <Td>
-                                    <Requester requester={requester} />
-                                </Td>
-                                <Td>
-                                    <RequestedAction exception={exception} context="CURRENT" />
-                                </Td>
-                                <Td>
-                                    <RequestCreatedAt createdAt={createdAt} />
-                                </Td>
-                                <Td>
-                                    <RequestExpires exception={exception} context="CURRENT" />
-                                </Td>
-                                <Td>
-                                    <RequestScope scope={scope} />
-                                </Td>
-                                <Td>
-                                    <RequestedItems exception={exception} context="CURRENT" />
-                                </Td>
-                            </Tr>
-                        );
-                    })}
+                    {tableUIState.type === 'LOADING' && (
+                        <Tr>
```

Comment:
I think as a follow up we should extract these table states into a top level component directory so that once we finalize the design and discuss with UX we can standardize across the app.